netstandard/pkg/NETStandard.Library.dependencies.props

```diff
@@ -144,7 +144,7 @@
       <TargetFramework>netcoreapp2.0</TargetFramework>
     </_dependency>
     <_dependency Include="$(PlatformPackageId)">
-      <TargetFramework>uap10.1</TargetFramework>
+      <TargetFramework>uap10.0.15138</TargetFramework>
```

Comment:
🙁 don't like the magic number in random source file.  Can you please look into centrally defining this and using a friendly property where needed?